### PR TITLE
#1620 french translation - initialization part

### PR DESCRIPTION
**Pull request by tkhadir**
#1620 french translation - initialisation part

- I have translated the main file README.md to /fr/README.md like others.
- #1620 

Pull request description

- I have added a french flag in the main README.md to be used as a link to the translation of it.
- The french flag is in assets/fr directory.
- I have translated the file README.md to french.
- The rest of patterns will be translated one part after the other
- I have added CN/KR flags for languages already presents and enabled links to them.

Thanks in advance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-148-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+<br/>
+
+Read in different language : [![FR](/assets/flags/FR.png)**FR**](/fr/README.md),
+
+<br/>
+
```
Comment (ohbus) [inline README.md:21]: If we plan to include this for French then we must include all other languages already available like Korean and Chinese

**Review by ohbus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ohbus) [reply README.md:21]: @iluwatar What are your suggestions on this?

Comment (tkhadir) [reply README.md:21]: hello @ohbus yes we must include chinese and korean in this part :)

Comment (tkhadir) [reply README.md]: done i have added chinese and korean flages in main README

**Review by ohbus — APPROVED** [review]
Thank you @tkhadir for your contribution.

File: README.md (hunk 1 above)
Comment (ohbus) [reply README.md]: Thank you.

**Comment by ohbus** [comment]
This PR partly addresses **#1620** as this is an ongoing process and @tkhadir will be working on [**this issue**](https://github.com/iluwatar/java-design-patterns/issues/1620) to make this translation process a success.

**Comment by ohbus** [comment]
@all-contributors please add @tkhadir for translation

**Comment by iluwatar** [comment]
Thanks @tkhadir, good work

**Comment by STudio26** [comment]
Thanks @tkhadir for the initial work. I submitted some improved translation for the french version.

**Pull request merged**